src/dropdown/list/dropdown-list.component.ts

```diff
@@ -49,12 +50,15 @@ import { ScrollableList } from "./../scrollable-list.directive";
 		<ul
 			#list
 			role="listbox"
-			class="bx--list-box__menu">
-			<li tabindex="{{item.disabled? -1 : 0}}"
+			class="bx--list-box__menu"
+			[attr.aria-label]="dropdownListLabel">
+			<li tabindex="-1"
 				role="option"
-				*ngFor="let item of displayItems"
+				*ngFor="let item of displayItems; let i = index"
 				(click)="doClick($event, item)"
 				(keydown)="doKeyDown($event, item)"
+				(focus)="listElementList[i].classList.add('bx--list-box__menu-item--highlighted'); listElementList[i].tabIndex = 0"
+				(blur)="listElementList[i].classList.remove('bx--list-box__menu-item--highlighted'); listElementList[i].tabIndex = -1"
```

Comment:
These focus/blur events are probably better wrapped up in a method ... something like `onItemFocus`